subprojects/packagefiles/capstone-bundled/meson.build

```diff
@@ -1,4 +1,4 @@
-project('capstone', 'c', meson_version: '>=0.55.0')
+project('capstone', 'c', meson_version: '>=0.55.0', default_options: ['werror=false'])
```

Comment:
was this due to some specific clang warning for capstone? I think we already compiled with werror with gcc and capstone-bundled builds fine there.

Reply:
Yes

![s_cpu_type](https://user-images.githubusercontent.com/12002672/143874002-826be615-507f-4b95-b3ce-cc1d3620c9d8.PNG)
(https://github.com/rizinorg/rizin/runs/4351274434?check_suite_focus=true#step:12:697)

Apparently that `s_cpu_type` variable is only used in a `CS_ASSERT()` call and therefore in a release build, the `CS_ASSERT()`s disappear and that variable is left hanging.

I would ofc prefer fixing that upstream but it can take time for such fixes to merge and any new Clang warnings that pop up in Rizin code should be caught immediately (such as the warning in #2042).

Reply:
> Apparently that `s_cpu_type` variable is only used in a `CS_ASSERT()` call and therefore in a release build, the `CS_ASSERT()`s disappear and that variable is left hanging.

Wait the above is not correct:

```c
if (M680X_CPU_TYPE_ENDING != ARR_SIZE(s_cpu_type)) {
    CS_ASSERT(M680X_CPU_TYPE_ENDING == ARR_SIZE(s_cpu_type));

    return CS_ERR_MODE;
}
```

Now the warning looks like a false positive so perhaps I should just disable the warning for Clang.

Reply:
Hmm now I think that Clang is justified in issuing that warning because the contents of `s_cpu_type` is not used.